akka-remote/src/main/scala/akka/remote/artery/compress/InboundCompressions.scala

```diff
@@ -128,6 +133,21 @@ private[remote] abstract class InboundCompression[T >: Null](
         activeVersion, nextTable.version, incomingTableVersion)
       OptionVal.None
     }
+  }
+
+  def confirmAdvertisement(tableVersion: Int): Unit = {
```

Comment:
Ah, right you even explained just that in the standup.
Would be nice to persist that reason in a comment here